projects/cds/src/profiletag/model/profile-tag.model.ts

```diff
@@ -26,3 +26,7 @@ export enum ProfileTagEventNames {
   ConsentReferenceChanged = 'ConsentReferenceChanged',
   ProfileTagDebug = 'DebugFlagChanged',
 }
+
+export interface ProfileTagPushEvent {
+  
+}
```

Comment:
Please add a new line at the end.
Also, make sure that all changed files are formatted correctly.